### PR TITLE
Use custom security.properties for SSL TLS1.0,1.1

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
jck api/javax_net tests TLS1.0 & 1.1 so must be enabled to support those tests.

Signed-off-by: Andrew Leonard <anleonar@redhat.com>

### CODE REVIEW

**Review by gdams — APPROVED** [review]

**Comment by andrew-m-leonard** [comment]
I've tested with jdk-11 and 16 as well

File: openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java [hunk 1]
```diff
@@ -324,6 +324,15 @@ public void setUp(StfCoreExtension test) throws Exception {
 		
 		test.doEchoFile("The JTB file generated\n", newJtbFileRef);
 
+		if ( tests.contains("api/javax_net") ) {
```
Comment (Mesbah-Alam) [inline openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java:327]: Just wondering, do these changes require any set up to be done somewhere else for them to work?

**Comment by andrew-m-leonard** [comment]
i’ve tested win & mac on jck8, 11 and 16

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
@andrew-m-leonard - could you please let us know if this fixes all the javax_net issues? 

This target has been disabled on 8 and 11 for ibm+openj9: https://github.com/adoptium/aqa-tests/blob/391642bdca37e09fdca74b0743353383915b337c/jck/runtime.api/playlist.xml#L935 

If this issue fixes all javax_net issues, we should un-exclude the target.

**Comment by smlambert** [comment]
We will not be able to confirm that @Mesbah-Alam, for 2 reasons
1) we do not have visibility of backlog/issues/461 
2) we are not allowed to verify this against non-Temurin builds.  I suggest that you check on an internal grinder by running BUILD_LIST=jck, TARGET=disabled.jck-runtime-api-javax_net now that this is merged

**Pull request merged**